using-timescaledb/continuous-aggregates.md

```diff
@@ -0,0 +1,249 @@
+# Continuous Aggregates 
+TimescaleDB continuous aggregates allow users to speed aggregate queries which
+span large amounts of time by continuously calculating them in the background
+and materializing the aggregates in a separate view. Conntinuous aggregates use
+the background worker framework to run aggregates on a schedule as new data is
+inserted into a hypertable or when old data is modified. There's a bit more
+complexity to how they're implemented which you can read about in our [blog post
+announcing the feature][continuous-aggs-blog], but for now, suffice it to say
```

Comment:
can we end up with a broken link in the docs? How long do we retain blog post links?

Reply:
not too worried, as we control our blog as well.